lib/Driver.js

```diff
@@ -139,9 +138,21 @@ module.exports = class Driver {
    *
    * @returns {string} Returns the key's id.
    */
-  async computeKeyId({key}) {
+  computeKeyId({key}) {
     return `did:key:${key.fingerprint()}#${key.fingerprint()}`;
   }
+
+  /**
+   * Generates a public verification method from a keypair.
+   *
+   * @param {LDKeyPair} key
```

Comment:
Should this be `keyPair`? Also, docs.

```suggestion
   * @param {LDKeyPair} key
```

Reply:
`key` has been used fairly heavily throughout the implementation when keyPair should probably have been used instead. Again, I was trying not to do major surgery and just add the things that I needed. I can do major surgery but it'll take away from doing other things. @dmitrizagidulin, thoughts?